procedure/pull-request-merge.rst

```diff
@@ -171,28 +188,44 @@ These steps will only rarely need repeating.
 #. Ensure your public SSH key is submitted in Perforce at
    //.git-fusion/users/USER/keys/
 
+   This is the public key that the SSH used by your Git will use to
+   connect to perforce.ravenbrook.com.
+
+   **NOTE**: As of 2022-11, you may not be able to connect to Perforce
+   Git Fusion without specifying an old public key algorithm.  See
+   `"Connecting to Git Fusion"
+   <https://info.ravenbrook.com/mail/2022/11/29/21-01-14/0/>`__.
+
 #. Ensure your e-mail address is submitted in Perforce at
    //.git-fusion/users/p4gf_usermap and matches your Perforce user
    record.
 
 #. Clone the Ravenbrook MPS GitHub repository and name the remote
-   "github".  This will give you access to CI to build and test the
-   merge.  (If you're an MPS developer you can use your existing
-   repo.)  ::
+   "github" [#github]_.  This will give you access to CI to
+   build and test the merge.  (If you're an MPS developer you can use
+   your existing repo.)  ::
 
      git clone -o github git@github.com:Ravenbrook/mps.git
-     cd mps
 
 #. Set your e-mail address for commits to the repo to match the one in
    your Perforce user record, e.g. ::
 
+     cd mps
```

Comment:
m: suggestion: From within the mps directory (you might need to "cd mps")

Reply:
Fix: Clarified in 28c0b07f7